torchvision/models/quantization/googlenet.py

```diff
@@ -5,14 +5,21 @@
 import torch.nn as nn
 from torch import Tensor
 from torch.nn import functional as F
-from torchvision.models.googlenet import GoogLeNetOutputs, BasicConv2d, Inception, InceptionAux, GoogLeNet, model_urls
+from torchvision.models.googlenet import GoogLeNetOutputs, BasicConv2d, Inception, InceptionAux, GoogLeNet
 
 from ..._internally_replaced_utils import load_state_dict_from_url
 from .utils import _fuse_modules, _replace_relu, quantize_model
 
 
 __all__ = ["QuantizableGoogLeNet", "googlenet"]
 
+
+model_urls = {
```

Comment:
Q: is this for BC? I thought we went for `_MODEL_URL` (or at least something private)?

Reply:
This is temporary so that the tests don't fail. I'm removing it on the next PR.